src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs

```diff
@@ -29,6 +29,6 @@ public class ListOptions : BaseOptions
         /// The order in which to paginate records.
         /// </summary>
         [JsonProperty("order")]
-        public PaginationOrder Order { get; set; } = PaginationOrder.Desc;
+        public PaginationOrder? Order { get; set; }
```

Comment:
Since we are trying to cut a major version, I'll add this change back in, as #136 intended.